.gitignore

```diff
@@ -9,4 +9,5 @@ node_modules
 telepresence.log
 venv
 *.swp
-tools/dev/cert/*
\ No newline at end of file
+tools/dev/cert/api_key_client
+tools/dev/cert/api_key_server
```

Comment:
Api key shouldn't be pushed to repo, because the api key is store in Dev container's database.